OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs

```diff
@@ -260,37 +261,67 @@ public async Task GetApprovedByWorkshopId_NoExistingChildren_ShouldReturnEmptySe
     //        Assert.ThrowsAsync<ArgumentException>(() => childService.UpdateChildForParetWithSpecifiedUserId(child));
     //    }
 
-    //    [Test]
-    //    public void ChildService_Delete_DeletesChild()
-    //    {
-    //        ChildDto child = new ChildDto { Id = 1, FirstName = "fn1", LastName = "ln1", MiddleName = "mn1", DateOfBirth = new DateTime(2003, 11, 9), Gender = Gender.Male, ParentId = 1, SocialGroupId = 2 };
-    //        mockRepository.Setup(m => m.Delete(It.IsAny<Child>()));
-    //        mockRepository.Setup(m => m.GetById(It.IsAny<long>())).Returns(Task.FromResult(GetTestChildEntities().First()));
+    [Test]
+    public async Task ChildService_DeleteWithTechAdminRole_DeletesChild()
+    {
+        // Arrange
+        var child = ChildGenerator.Generate().WithGeneratedParent();
 
-    //        IChildService childService = new ChildService(mockRepository.Object, logger.Object, localizer.Object);
+        var childList = new List<Child> { child }.BuildMock();
 
-    //        childService.DeleteChildCheckingItsUserIdProperty(1);
+        childRepositoryMock.Setup(m => m.GetByFilterNoTracking(It.IsAny<Expression<Func<Child, bool>>>(), It.IsAny<string>()))
```

Comment:
(Clear test code and It.IsAny) The specific `It.IsAny<string>()` here might be not any string, but `nameof(Child.Parent)`. So we are sure, that `childService` code is working with right filter. Please consider changes in all your tests.